manifests/r/rgriebl/brickstore/2022.4.1.0/rgriebl.brickstore.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: rgriebl.brickstore
+PackageVersion: 2022.4.1.0

```

This must match control panel. When changing this value, don't forget to change the name of the version folder
```suggestion
PackageVersion: 2022.4.1
```